src/main/scala/com/raphtory/core/deploy/RaphtoryService.scala

```diff
@@ -9,6 +9,46 @@ import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe.TypeTag
 
+/**
+  * {s}`RaphtoryService[T]`
+  *
+  *  : `RaphtoryService` for initialising spout, graphbuilder, deploying spout, partition and query manager
+  *
+  * ## Methods
+  *
+  *    {s}`defineSpout(): Spout[T]`
+  *      : Initialise Spout
```

Comment:
this is an abstract method, any examples of how it is overridden when the class is extended? for example you could say the defineSpout is a function that is overriden by objects that extend the raphtory service, this is used to construct the spout which is used to ingest data. an example of ovveriding this fn is as follows. e.g.